index.js

```diff
@@ -1 +1,22 @@
 console.log("Code your solution!")
+
+const input = document.querySelector('input')
+const button = document.querySelector('button')
+const form = document.querySelector('form')
+
+form.addEventListener('submit', event => {
+    event.preventDefault()
+    console.log(input.value)
+    const li = document.createElement('li')
+    li.textContent = input.value
+    document.querySelector('ul').append(li)
+
+    const list = document.querySelectorAll('ul li')
+
+    list.forEach(li => {
+        li.addEventListener('click', () => {
+            li.setAttribute('style', 'text-decoration: line-through')
```

Comment:
This works too, but there's a simpler, cleaner, and more standard method: `.style`. Try:

`li.style.textDecoration = "line-through";`